charts/mokta/templates/deployment.yaml

```diff
@@ -0,0 +1,41 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mokta
+  annotations:
+{{- if .Values.annotations }}
+{{ toYaml .Values.annotations | indent 8 }}
+{{- end }}
+  labels:
+    {{- include "labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "selectorLabels" . | nindent 6 }}
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        {{- include "selectorLabels" . | nindent 8 }}
+        app.kubernetes.io/name: {{ .Release.Name }}
+    spec:
+      # serviceAccountName: mokta
+      securityContext:
+        runAsUser: 0
+        runAsGroup: 0
+      containers:
+      - name: mokta
+        volumeMounts:
+          - name: okta-form
+            mountPath: /app/form
+        env:
+          {{- range $key, $val := .Values.env }}
+          - name: {{ $key }}
+            value: {{ $val | quote }}
+          {{- end }}
+        image: {{ .Values.image }}
```

Comment:
not sure if we want to do it, but in charts generated by `helm create`, the image is split into `image.repository` and `image.tag` and the tag defaults to the `appVersion`, so that you can control the version that is deployed directly in `Chart.yaml`

Reply:
That makes sense, I'll change it up